### PR TITLE
add projection extension metadata to GRD Items

### PR DESCRIPTION
**Pull request by philvarner**
**Related Issue(s):**

n/a

**Description:**

- add projection extension metadata to GRD Items

**PR checklist:**

- [X] Code is formatted (run `scripts/format`).
- [X] Code lints properly (run `scripts/lint`).
- [X] Tests pass (run `scripts/test`).
- [X] Documentation has been updated to reflect changes, if applicable.
- [X] Examples have been updated to reflect changes, if applicable
- [X] Changes are added to the [CHANGELOG](../CHANGELOG.md).

### CODE REVIEW

**Review by pjhartzell — APPROVED** [review]

**Review by pholleway — APPROVED** [review]

**Review by gadomski — CHANGES_REQUESTED** [review]
Needs an `examples/` update?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -31,4 +31,4 @@ repos:
       rev: v0.33.0
       hooks:
           - id: markdownlint
-            args: ["--ignore", ".github/ISSUE_TEMPLATE/feature_request.md"]
+            args: ["--ignore", ".github/ISSUE_TEMPLATE/feature_request.md", "--disable", "MD024"]
```
Comment (philvarner) [inline .pre-commit-config.yaml:34]: disables duplicate header detection, so the README can have multiple subheadings named RTC and GRD.

File: README.md [hunk 2]
```diff
@@ -43,12 +43,18 @@ Level-1](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/resolu
 product. It is used to create STAC Items from the SAFE manifest format of the
 data hosted on Microsoft Azure.
 
-## Examples
+## Example Outputs
 
-### STAC objects
+### RTC
 
-- [Item (RTC)](examples/sentinel1-rtc-aws/2016/S1B_20161121_12SYJ_ASC/S1B_20161121_12SYJ_ASC.json)
```
Comment (philvarner) [inline README.md:50]: this were all broken

File: CHANGELOG.md [hunk 3]
```diff
@@ -24,6 +28,7 @@ number as needed.
 ### Fixed
 
 - Fixed updated item-asset titles to work with COG format granules. ([#37](https://github.com/stactools-packages/sentinel1/pull/37))
+>>>>>>> main
```
Comment (gadomski) [inline CHANGELOG.md:31]:
```suggestion
```

Leftover merge conflict marker.

**Review by gadomski — CHANGES_REQUESTED** [review]

**Review by gadomski — APPROVED** [review]

**Pull request merged**